### PR TITLE
Feature/deeplinking

### PR DESCRIPTION
**Pull request by totorototo**
work in progress. do not merge right now! 
Authenticate saga not finished yet. @guijesuis

### CODE REVIEW

File: app/deeplink.js [hunk 1]
```diff
@@ -8,7 +8,14 @@ export function subcribe(store) {
   function handleOpenURL(url) {
     if (url != null && url.startsWith('strava://')) {
       const relativeUrl = url.substring(Platform.OS === 'android' ? 'strava://strava/'.length : 'strava;//'.length);
-      store.dispatch(NavigationActions.navigate({ routeName: 'localhost', params: {state: '', code: '31305d0d2312fc4d95479f54c53b16ba486f84c9'} }));
+      const tokenRegexp = /(?:&code=)(\w*)/g;
```
Comment (guijesuis) [inline app/deeplink.js]: no!!!

Comment (guijesuis) [reply app/deeplink.js]: we should parse query -> no specific here

**Review by guijesuis — CHANGES_REQUESTED** [review]

File: app/deeplink.js (hunk 1 above)
Comment (totorototo) [reply app/deeplink.js]: got your point!

**Comment by totorototo** [comment]
changes done! to be merged after being reviewed.

**Comment by totorototo** [comment]
waiting for your approval, Rick!

**Pull request merged**